tests/vars.sample.yaml

```diff
@@ -61,3 +61,11 @@ compute_adoption: true
 
 # Where perform or not telemetry installation during adoption
 telemetry_adoption: true
+# Whether source env is OSPD Director Operator
+# i.e. https://github.com/openstack-k8s-operators/osp-director-operator
+ospdo_src: false
+
+# rhoso namespace
+osp18_namespace: "openstack"
+# director operator namespace
+org_namespace: "ospdo_openstack"
```

Comment:
I just realized these variables should be better named as well. Your comments are clear on the intended use, but the variable names themselves aren't. There's probably no reason why the variable names can't be clear too?

Let's do this instead:
```
rhoso_namespace: "openstack"
# ospdo_namespace: "ospdo_openstack"
```

The`ospdo_namespace` should remain unset by default, or it will break the default use case. When we're doing `{{ ospdo_namespace| default('openstack') }}` we want it to evaluate to 'openstack' 99% of the time, only the ospdo job is an exception.

Reply:
changed the osp18_namespace -> rhoso_namespace
but the not sure about the use of "ospdo_namespace" as apposed to org_namespace
most of the use of this automation would be for non ospdo envs. so adding the "ospdo" name , 
in multiple places in the code would be confusing no ? since most users & future users might not get what the "ospdo" wording is about.. so that org_namespace would give the simplest explanation ?
since ospdo would be much less used..?